sonic_platform_base/psu_base.py

```diff
@@ -223,6 +223,26 @@ def get_maximum_supplied_power(self):
         """
         raise NotImplementedError
 
+    def get_psu_power_warning_threshold(self):
+        """
+        Retrieve the warning threshold of the power on this PSU
```

Comment:
Need one API so that user can override the warning suppress level

Reply:
As discussed offline, we do not need a platform API to do it because platform API can not store the information.